### PR TITLE
Add oscap-anaconda-addon builds to Jenkins

### PR DESCRIPTION
**Pull request by redhatrises**
- Adding oscap-anaconda-addon rhel7 and master branch builds to jenkins to validate the builds and prevent the breaking of tests

### CODE REVIEW

**Comment by mpreisler** [comment]
Wow, good stuff. How did you create this?

**Comment by redhatrises** [comment]
@mpreisler magic!

**Comment by redhatrises** [comment]
@mpreisler unless I am misunderstanding what you are asking, I just used the existing content in the `ansible` dir in this repo. Of course, someone such as yourself would have to run it for Jenkins as well as add in the github hooks.

BTW. A lot of the tests are broken on OAA, so adding this to Jenkins we can get some sort of testing whenever some makes a change unless it is something new.

**Comment by mpreisler** [comment]
> @mpreisler magic!

burn the heretic!

So you wrote this XML by hand and never tested it? @dahaic  created the other XMLs by clicking around in Jenkins and then exporting them as XML. AFAIK...

**Comment by redhatrises** [comment]
>> @mpreisler magic!

> burn the heretic!

hahaha.... Man... that's twice this week from separate people.

> So you wrote this XML by hand and never tested it?

More like copied and modified from the existing XMLs. I am not sure that I can really test them. I will try what @dahaic did. See if there are any differences.

**Comment by redhatrises** [comment]
@mpreisler just now generated the XMLs similar to the ones that I have in this PR.

**Comment by dahaic** [comment]
Ok, I am now confused - are the XMLs in the PR those we want to commit? :)

Mini Review:

RHEL7 does not have python2-mock, you need EPEL. Thus it's necessary to move epel enabling from master to shared.

As I have not noticed this PR, I have made my own fix of ansible, feel free to use this patch, either for cherry-pick or as an inspiration :) https://github.com/dahaic/jenkins/commit/3c0ecbad2261fefe37d4e6b0b96087b4a3b56a42

I am not sure about python3 packages though, as I have actually let ansible perform changes in my commit, and fedora works just fine. (well, job is still failing, but for different reason)

File: ansible/workers.yml [hunk 1]
```diff
@@ -134,3 +134,27 @@
...
+  - name: Ensure OSCAP Anaconda dependencies are installed (RHEL7 only)
+    package:
+      name:
+      - anaconda
+      - python2-mock
+      - python-nose
+      - python-cpio
+      - zanata-python-client
+      state: installed
+    when: ansible_distribution == 'RedHat' or ansible_distribution_version.split(".")[0] >= "7"
```
Comment (dahaic) [inline ansible/workers.yml]: `and` not `or` It triggers RHEL6 as well.

File: ansible/workers.yml [hunk 2]
```diff
@@ -134,3 +134,27 @@
...
+
+  - name: Ensure OpenSCAP Anaconda dependencies are installed (Fedora only)
+    package:
+      name:
+      - anaconda
+      - python3-mock
+      - python3-nose
+      - python3-cpio
+      - python3-kickstart
+      - zanata-python-client
```
Comment (dahaic) [inline ansible/workers.yml]: Can you add `python3-pycurl` ? It is required as well. See

https://jenkins.open-scap.org/job/oscap-anaconda-addon-pull-requests/distributions=fedora/10/console

[Note, I already installed it on fedora machine - now it fails on something else again...]
https://jenkins.open-scap.org/job/oscap-anaconda-addon-pull-requests/distributions=fedora/11/console

But that is not in the scope of Jenkins configuration.

**Review by dahaic — COMMENTED** [review]
Apart from these last notes, I think it's OK for merge.

**Comment by dahaic** [comment]
ACK, thank you for this!

**Comment by redhatrises** [comment]
@dahaic you're welcome. Thanks for review and helping!

**Pull request merged**